src/ResultSetBuilder.php

```diff
@@ -59,7 +59,7 @@ public function buildModelFromIndexAndData(string $indexName, $source)
         return $this->buildModel($indexName, $source, []);
     }
 
-    public function buildModelFromDocument(Document $document)
+    public function buildModelFromDocument(ElasticaDocument $document)
```

Comment:
This is a BC break. We should revert this line, and deal with both class. But I guess there is nothing to do apart reverting this line

Reply:
No it's not :thinking: it's the same signature, only the imported named changed :thinking:

Reply:
Oups 🤦🏼 Thanks